### PR TITLE
Feature/add user fields for local admin

### PR DESCRIPTION
**Pull request by appsol**
### Summary
https://trello.com/c/Y4tWnI63

- Create local_authorities table and migration
- Create LocalAuthority Model and Controler
- Create LocalAuthoritySeeder to fetch jsondata from .gov.uk site and incrementally populate table (Link for data is stored in .env file)
- Add User->employer_name
- Add User->location_id and Location relationship
- Add User->local_authority_id and LocalAuthority relationship

### Development checklist
- [x] Changes have been made to the API?
  - [x] If so, the OpenAPI specification has been updated
- [x] Database migrations have been added?
  - [x] If so, the MySQL Workbench ERD has been updated
- [x] The code has been linted `./develop composer fix:style`

### Release checklist
- [ ] Run `hlp:la-import http://geoportal1-ons.opendata.arcgis.com/datasets/fe6bcee87d95476abc84e194fe088abb_0.geojson` after migration

### Notes
Current URL to use to fetch data: http://geoportal1-ons.opendata.arcgis.com/datasets/fe6bcee87d95476abc84e194fe088abb_0.geojson

### CODE REVIEW

**Review by matthew-inamdar — COMMENTED** [review]
I'm unable to fully test as I'm not sure what value to use for `LOCAL_AUTHORITY_DATA_URL`?

Could we also add this value to the `.env.example` file? If there isn't a sensible default, then an empty value is fine.

File: database/seeds/LocalAuthoritySeeder.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
```
[the remark this answers is not in the document]
Comment (appsol) [reply database/seeds/LocalAuthoritySeeder.php:1]: I left the URL in the PR notes, but you might have missed it.
I'll look at turning this into an Artisan command tomorrow.

[the remark this answers is not in the document]
Comment (appsol) [reply database/seeds/LocalAuthoritySeeder.php]: - Refactored seeder into command.
- Merged in develop and resolved conflicts.

**Review by matthew-inamdar — APPROVED** [review]
LGTM 👍

**Pull request merged**